.github/workflows/pythonpublish.yml

```diff
@@ -32,16 +34,38 @@ jobs:
         run: |
           python setup.py sdist bdist_wheel
           twine upload dist/*
-      - name: Autobump plugin version
```

Comment:
@evalsocket why did this disappear? i think we need this to bump plugin versions on top of the master version in line 22